torchaudio/_extension.py

```diff
@@ -103,12 +97,15 @@ def _init_extension():
                     pass
 
     _load_lib("libtorchaudio")
-    # This import is for initializing the methods registered via PyBind11
-    # This has to happen after the base library is loaded
-    from torchaudio import _torchaudio  # noqa
 
     # Because this part is executed as part of `import torchaudio`, we ignore the
     # initialization failure.
+    _load_lib("libtorchaudio_sox")
```

Comment:
since including sox is not compulsory, should this be moved to the try-catch block below so that failing to load `libtorchaudio_sox` doesn't prevent importing `torchaudio` altogether?

Reply:
When we move to dynamic linking of libsox, that will be absolutely necessary. Right now, it is not necessary but I can add it. Let me update the code.